pkg/testkit/mocksessionmanager.go

```diff
@@ -158,8 +158,8 @@ func (msm *MockSessionManager) GetInternalSessionStartTSList() []uint64 {
 			txn := se.TxnInfo()
```

Comment:
As it also used `TxnInfo`, should we also modify it to align with the logic in `GetStartTSFromSession`?